### PR TITLE
Upgrade deps: sbt and play-ebean

### PR DESCRIPTION
**Pull request by bion**
Upgrades:

- [x] sbt (the project's build tool) upgraded 1.3.13 -> 1.6.2
- [x] play-ebean (the plugin that integrates ebean into Play) upgraded 6.0.0 -> 6.2.0-RC4
- [x] ebean (the project's ORM) upgraded 11.45.1 -> 12.8.1 (implicitly via [play-ebean](https://github.com/playframework/play-ebean#releases))

SBT versions higher than 1.3.13 (current project version) are not compatible with play-ebean 6.0.0 (current project version) so these need to be upgraded together.

outstanding issues:

- [x] fix the ebean deprecation warnings
- [x] fix compile loop problem

closes https://github.com/seattle-uat/civiform/issues/1942

### CODE REVIEW

**Comment by shanemc-goog** [comment]
LGTM but may want to hold off pending the prod deployment this week

**Review by shanemc-goog — APPROVED** [review]

File: universal-application-tool-0.0.1/project/plugins.sbt
Comment (dcsobral) [inline universal-application-tool-0.0.1/project/plugins.sbt:3]: Is this change required? Both the sbt version change and this are the ones that seem most likely to be associated with the build problem, so doing only one of them would be good for diagnosis.

Comment (bion) [reply universal-application-tool-0.0.1/project/plugins.sbt:3]: Unfortunately they need to be made together. The newer `play-ebean` versions don't work with sbt 1.3.13 and require at least 1.4y. That being said it may be worth trying a smaller jump for sbt than 1.6.1.

File: .github/workflows/pr.yaml
Comment (shanemc-goog) [inline .github/workflows/pr.yaml:16]: this would appear to be redundant with the env right above... Is this WAI and just unfamiliar to me or odd enough we should comment it?

**Review by shanemc-goog — APPROVED** [review]

File: .github/workflows/pr.yaml
Comment (bion) [reply .github/workflows/pr.yaml:16]: done

**Pull request merged**
